editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
Yes, this is something I came up with after reading the second part here https://github.com/reactjs/reselect#q-how-do-i-create-a-selector-that-takes-an-argument